src/trace/README.md

```diff
@@ -0,0 +1,92 @@
+## Introduction
+
+This directory contains the implementation of tracing using [LTTng](https://lttng.org/) (Linux Trace Toolkit Next Generation).
+
+## LTTng Installation
+
+To install LTTng on your Linux system, follow the instructions provided in the [LTTng documentation](https://lttng.org/download/)
+
+> Dependency LTTNG version is greater than 2.12.
+
+## LTTng QuickStart
+
+LTTng is an open source tracing framework for Linux that provides highly efficient and low-overhead tracing capabilities. It allows developers to trace both kernel and user-space applications.
+
+Building Valkey with LTTng support:
+
+```
+USE_LTTNG=yes make
+```
+
+Enable lttng trace events dynamically:
+```
+~# lttng destroy valkey
+~# lttng create valkey
+~# lttng enable-event -u "valkey*"
```

Comment:
We typically use single quotes to make sure the globbing pattern is not expanded, e.g.

lttng enable-eveent -u 'valkey*'